src/utils/backend.js

```diff
@@ -169,3 +173,13 @@ export default class Backend {
     headers: { 'Content-Type': 'application/json' },
   });
 }
+
+export const issueReport = (report) => {
```

Comment:
won't be consistent to define it as a static method of the above class?

Reply:
right, done